### PR TITLE
Дмитричев - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by DmitrichevNick**

### CODE REVIEW

**Comment by DmitrichevNick** [comment]
Can you guys review this? :poop:
@Marat-Israfilov
@GorozhaninMihail

**Comment by Marat-Israfilov** [comment]
:+1:

File: code/range/model/range.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            self.__start_value += 1
+        if range_str.endswith(')'):
+            self.__end_value -= 1
+        if self.__end_value < self.__start_value:
+            raise ValueError('Logic error: no numbers in range')
+        self.__range_str = range_str
+
+    def contains_value(self, obj):
+        if not isinstance(obj, int):
+            raise TypeError('Wrong type of obj')
```
Comment (Marat-Israfilov) [inline code/range/model/range.py:23]: I think, you need to add expected type in exception message

**Comment by Marat-Israfilov** [comment]
👍

**Comment by GorozhaninMihail** [comment]
👍

**Comment by DmitrichevNick** [comment]
@kirill-korniakov

**Review by sawirani — APPROVED** [review]

**Pull request merged**